CODEOWNERS

```diff
@@ -0,0 +1,42 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+# For supported syntax see docs:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# Default owners if there is no more specific rule.
+*   @ShyamsundarR @BenamarMk
+
+# volrep
+# TODO: second owner
+/internal/controllers/vrg_volrep*.go    @ShyamsundarR
+
+# volsync
+# TODO: second owner
+/internal/controllers/volsync/          @BenamarMk
+/internal/controllers/vrg_volsync*.go   @BenamarMk
+
+# cephfs consistency groups
+# TODO: second owner
+/internal/controllers/cephfscg/     @BenamarMk
+
+# Metrics
+# TODO: second owner
+/docs/metrics.md                    @rakeshgm
```

Comment:
@rakeshgm does not have write access yet so github complains.

Reply:
Fixed for @rakeshgm and @ELENAGER